frontend/src/core/comments/components/AddComment.js

```diff
@@ -25,69 +34,87 @@ export default function AddComments(props: Props) {
         addComment,
     } = props;
 
-    const commentInput: any = React.useRef();
-    const minRows = 1;
-    const maxRows = 6;
+    let [editorState, setEditorState] = React.useState(
+        EditorState.createEmpty()
+    )
+    const editor: any = React.useRef(null);
+    const rawContentState = convertToRaw(editorState.getCurrentContent());
```

Comment:
Should we only do the conversion before saving?

Reply:
Yes. I have moved it into the submit function.